app/controllers/bridges/lom/tasks_controller.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+module Bridges
+  module Lom
+    # rubocop:disable Metrics/ClassLength
+    class TasksController < ActionController::API
+      OML_SCHEMA_PATH = Rails.root.join('vendor/assets/schemas/lom_1484.12.3-2020/lom.xsd')
+
+      def show
+        builder = Nokogiri::XML::Builder.new(encoding: 'UTF-8') { |xml| sample_oml(xml) }
+
+        if params[:validate].present?
+          schema = Nokogiri::XML::Schema(File.read(OML_SCHEMA_PATH))
+          errors = schema.validate(builder.doc)
+
+          return render plain: errors.map(&:message).join("\n") if errors.any?
+        end
```

Comment:
I really like the possibility to validate the resulting XML against the XSD. Super handy :+1: